idl/lpc55-update.idol

```diff
@@ -59,7 +59,7 @@ Interface(
             encoding: Hubpack
         ),
         "status": (
-            doc: "Get info about installed images",
+            doc: "Get info about installed images (deprecated - use rot_boot_info)",
```

Comment:
Could this just be removed, since it's an IPC rather than a network call?

Reply:
As discussed in chat, these calls are used by SpRot and we have to be careful about removing them.